RIOT-OS/RIOT
# sys/registry: Add EEPROM storage

### Contribution description
This PR adds an EEPROM based storage facility for the RIOT Registry (#10799). This allows to save the configuration parameters in an EEPROM memory. For handling the data it uses the eepreg module.

It also modifies the `tests/registry` application to easily integrate this storage.

### Testing procedure
Modify the Makefile of the `tests/registry` application to use the `registry_store_eeprom` instead of the `registry_store_dummy`. Run the application, modify the values and save them. They should persist after rebooting the node.

### Issues/PRs references
Depends on PR #10799.
